drivers/watchdog/wdt_mchp_xec.h

```diff
@@ -0,0 +1,55 @@
+/* wdt_xec.h - Microchip XEC watchdog driver private definitions */
+
+/*
+ * Copyright (c) 2019 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_WDT_WDT_XEC_H_
+#define ZEPHYR_DRIVERS_WDT_WDT_XEC_H_
+
+#ifdef __cplusplus
```

Comment:
just curious, when would this header ever get compiled by c++ code?

Reply:
copy/paste superfluous stuff indeed